src/modules/md/md.cpp

```diff
@@ -48,6 +49,14 @@ MDModule::MDModule() : Module("MD")
         "Only forces arising from intramolecular terms (including pair potential contributions) will be calculated",
         intramolecularForcesOnly_);
 
+    // Output
+    keywords_.add<OptionalIntegerKeyword>("Output", "EnergyFrequency", "Frequency at which to calculate total system energy",
```

Comment:
Just the initial argument needs removing - this was the old way of specifying the group the keyword would be displayed in in the GUI, but this is now covered by the `setOrganisation()` method.
```suggestion
    keywords_.setOrganisation("Output");
    keywords_.add<OptionalIntegerKeyword>("EnergyFrequency", "Frequency at which to calculate total system energy",
```
etc.